packages/@o3r/rules-engine/src/engine/operator/facts/current-time/current-time-facts.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { FactsService } from '../../../../fact';
+import { RulesEngineService } from '../../../../services';
+import { CurrentTimeFacts } from './current-time.facts';
+
+@Injectable({
```

Comment:
Do we really want to be Angular injectable fact in the case of engine inner facts

Reply:
you have to register it in the app like any other fact service